src/test/java/Tests/object/BCommentTable.java

```diff
@@ -62,6 +74,22 @@ public boolean isCommentInactive(String state) {
         }
     }
 
-}
-
+    @Step("Verify presence of the category in comment categories ")
+    public boolean verifyCategories(String expected) throws NoSuchElementException {
+        try {
+            List<WebElement> categories = this.driver.findElements(By.xpath("//*[@class='categorycolumn']"));
+            for (WebElement row : categories) {
+                int i = 0;
```

Comment:
@lvrub why are you using `i` here? I believe this and next line has to be deleted.

Reply:
fixed